### DISCUSSION TITLE
How to style a produced matplotlib image

### DISCUSSION THREAD

**Discussion by blaylockbk** [pyscript/pyscript#1502] [Q&A]
Hi,

I have a page that creates and displays a matplotlib image

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
    <script defer src="https://pyscript.net/latest/pyscript.js"></script>
  </head>

  <body>
    <py-config>
      packages = ["matplotlib"]
    </py-config>

    <div class="container">

      <h3>Show Graph</h3>
      <div id="graph-area"></div>

    </div>

    <py-script>
      import matplotlib.pyplot as plt
      plt.plot([1,2,3])
      plt.tight_layout()
      display(plt, target="graph-area", append=False)
    </py-script>
  </body>
</html>

```

How can I add style the image that is produced, (e.g., change the max-width to 70%, or add a drop shadow).

<img width="935" alt="image" src="https://github.com/pyscript/pyscript/assets/6249613/be5c29ea-84b0-4ae3-ba36-a455a1e67875">

Sorry for asking something that I think has a very simple answer; I know python much better than I know html/css.

**Comment by ccalvocm** [comment]
Greetings @blaylockbk 

You can style the image that is produced by adding CSS rules to the #graph-area element. Here's an example that sets the maximum width to 70% and adds a drop shadow:
```

#graph-area {
  max-width: 70%;
  box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.3);
}
```

You can add these CSS rules to the <head> element of your HTML document, like this:
```

<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
  <style>
    #graph-area {
      max-width: 70%;
      box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.3);
    }
  </style>
  <script defer src="https://pyscript.net/latest/pyscript.js"></script>
</head>
```

This will apply the CSS rules to the #graph-area element and style the image that is produced by Matplotlib. Note that you can adjust the values of the max-width and box-shadow properties to achieve the desired effect.
